<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Інтегрування</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: italic 20pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>∛th(x) dx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 26pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>∫</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="b">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.989999999999998</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="a">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.989999999999998</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Метод прямокутників</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Метод трапецій</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Метод Сімпсона</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="eps">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>22</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>0.990000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>ε =</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableRiemann">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>xᵢ</string>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yᵢ</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>f(xᵢ + h/2)</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableSimpson">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>xᵢ</string>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yᵢ</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableTrapezoidal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>xᵢ</string>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yᵢ</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>Порахувати</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>I = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>I = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>I = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>Праві:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>Ліві:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>Середні:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="integralRight">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="integralLeft">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="integralMidpoint">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="integralSimpson">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>I = </string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>90</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>R(f) = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorRiemannRight">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>R(f) = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorSimpson">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>R(f) = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorTrapezoidal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="integralTrapezoidal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorRiemannMidpoint">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorRiemannLeft">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 14pt &quot;Segoe UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>I = </string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>20</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Введіть межі інтегрування та похибку:</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
